java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -30,7 +30,6 @@ cluster.path=\
 disabled.modules=\
```

Comment:
So many changes! However most of them are just reorderings, right? Hard to guess what was the real intent, can you revert the changes and start once again and create a minimal diff?

Reply:
OK, now it is better. Adding:
```
org.netbeans.modules.db.core
org.netbeans.modules.db.dataview
```
seems sane. Not sure why we need `com.google.guava`!?